### PR TITLE
bc 1.0.6 (new formula)

### PR DESCRIPTION
**Pull request by sjackman**

### CODE REVIEW

**Comment by dunn** [comment]
`keg_only :provided_by_osx`?

File: bc.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class Bc < Formula
+  desc "arbitrary precision numeric processing language"
+  homepage "https://www.gnu.org/software/bc/"
+
```
Comment (DomT4) [inline bc.rb]: This empty line might well upset the bottle placement logic, heh.

File: bc.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  url "http://ftpmirror.gnu.org/bc/bc-1.06.tar.gz"
```
Comment (DomT4) [inline bc.rb]: Can you add the `mirror "https://ftp.gnu.org/` style mirror as well please? At some point we'll be offering people a fallback from insecure urls to secure mirrors.

**Comment by sjackman** [comment]
Done. Thanks for the feedback, Dominyk.

File: bc.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      "--disable-silent-rules",
+      "--prefix=#{prefix}",
+      "--infodir=#{info}",
+      "--mandir=#{man}"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/bc", "--version"
+    assert_match "2", shell_output("echo 1+1 |#{bin}/bc")
```
Comment (bfontaine) [inline bc.rb]: You can use `pipe_output("#{bin}/bc", "1+1\n")` here.

Comment (sjackman) [inline bc.rb]: Done.

**Comment by bfontaine** [comment]
Merged; thank you @sjackman! :tada:

**Comment by sjackman** [comment]
Thanks for merging, Baptiste!

**Pull request closed without merging**